api/internal.go

```diff
@@ -31,6 +35,7 @@ type PartyInfo struct {
 	recipients map[[nacl.KeySize]byte]string  // public key -> URL
 	parties    map[string]bool  // Node (or party) URLs
 	client     utils.HttpClient
+	grpc string
```

Comment:
what's the gRPC parameter for here?

Reply:
Using this string to maintain the address on which the gRPC is running, instead of passing gRPC parameter along every function.